agent/agent_worker.go

```diff
@@ -493,12 +493,12 @@ func (a *AgentWorker) AcceptAndRunJob(job *api.Job) error {
 	return a.RunJob(accepted)
 }
 
-func (a *AgentWorker) RunJob(job *api.Job) error {
+func (a *AgentWorker) RunJob(acceptResponse *api.Job) error {
```

Comment:
If one was being pedantic, this could also be an `acquireResponse`. But they have exactly the same format and I still agree this name is better than just `job`.